.github/workflows/build.yml

```diff
@@ -69,15 +68,18 @@ jobs:
         sed -i '/version: git/d' snapcraft.yaml
         yq -i 'del(.parts.cleanup)' snapcraft.yaml
       if: matrix.arch == 'aarch64'
-    - uses: diddlesnaps/snapcraft-multiarch-action@v1
-      if: matrix.arch == 'aarch64'
+    - uses: diddledani/snapcraft-multiarch-action@v1
```

Comment:
The action is still under `diddlesnaps/snapcraft-multiarch-action`

Reply:
Ooo sorry, I mistakenly changed it.